### ISSUE TITLE
Installation of Open Cpu on RHEL 6.4

### ISSUE THREAD

**Issue by raghu557** [Microsoft/microsoft-r-open#43]
Hello,

We have oopencpu on RHEL Server 6.4, While installing everything got installed without any errors, but when we try to see the libraries of the open cpu using curl http://localhost/ocpu/library/ we are unable to see any response.

Can you please help us resolving this.

Regards,
Raghuram.

**Comment by raghu557** [comment]
Hello Team,

We have MRO-3.3 and R-devel 3.4.1 with rApache 1.7 and opencpu-1.67.  Please let us if we are missing anything here. 

We followed the below link for the installation
https://github.com/opencpu/opencpu-server/blob/master/rpm/buildscript.sh

Please help us in resolving this and quick help is much appreciated.

Regards,
Raghuram.

**Comment by mjmg** [comment]
You have installed two conflicting packages: MRO-3.3 and R-devel 3.4.1

MRO rpms already provide R-devel file dependecies although they are not declared properly. The script probably assumes a CRAN/"vanilla" R installation but MRO is packaged differently.

I solved this conundrum by creating and installing a "fake" R packages spec file which enumerates the provides properly. 
https://gist.github.com/mjmg/5a48c7f8ba2e48f3931b4d92000b76d0

I also installed rApache and opencpu from source rpms so dependencies are properly installed with yum-buildddep.
http://download.opensuse.org/repositories/home:/jeroenooms:/opencpu-2.0/Fedora_26/src/
Dont forget to fix SELinux permissions after installing rpm packages.
